Frontend/Pages/UserManagement/Login.razor

```diff
@@ -52,8 +58,37 @@
 @code {
     readonly Models.DTO.Login _login = new();
 
-    private void HandleValidSubmit() {
-        throw new NotImplementedException();
-    }
+    private async void HandleValidSubmit() {
+        var request = new HttpRequestMessage(HttpMethod.Post,
+            "https://localhost:7038/api/login");
+
+        request.Content = new StringContent(JsonConvert.SerializeObject(_login), Encoding.UTF8, "application/json");
+        
+        var client = ClientFactory.CreateClient();
+
+        var response = await client.SendAsync(request);
 
+        if (response.IsSuccessStatusCode)
+        {
+            NavManager.NavigateTo("/");
+        }
+        else
+        {
+
+            var statusCode = response.StatusCode;
+
+            switch (statusCode)
+            {
+                case HttpStatusCode.OK:
+                    NavManager.NavigateTo("/register");
+                    break;
+                case HttpStatusCode.Unauthorized:
+                    NavManager.NavigateTo("/unauthorized");
+                    break;
+                default:
+                    NavManager.NavigateTo("/500");
```

Comment:
Inconsistent, should be "/internal-server-error"

Reply:
I agree. Though I dont really understand the semantics behind this switch anyway. Status Code OK cannot be reached and even if, why would you redirect to register? Shouldn't there be an error message when entering invalid credentials instead of a redirection to a different page?